tools/HelpGeneration/HelpGeneration.psm1

```diff
@@ -209,12 +209,17 @@ function Validate-MarkdownHelp
                             $fileErrors += "No examples found"
                         }
                     }
-                    "*@{Text=}*"
+                    "@{Text=}"
                     {
                         # This case occurs when there is no description provided for a parameter
                         $parameter = $content[$idx-1].Substring(5)
                         $fileErrors += "No description found for parameter $parameter"
                     }
+                    ".``````yaml"
```

Comment:
Maybe instead of "." search for ![\n] - not exactly sure how regex works in powershell

Reply:
This is a good reference: http://www.powershellcookbook.com/recipe/qAxK/appendix-b-regular-expression-reference

The not in a character class is pretty much the same as in c# (caret)